EIPS/eip-2767.md

```diff
@@ -0,0 +1,239 @@
+
+Traditionally, many contracts that require that they be owned or controlled in some way use `ERC-173` which standardized the use of ownership in the smart contracts. For example to withdraw funds or perform administrative actions.
+
+```solidity
+contract dApp {
+  function doSomethingAdministrative() external onlyOwner {
+    // admin logic that can be performed by a single wallet
+  }
+}
+```
+
+Often, such administrative rights for a contract are written for maintenance purpose but users need to trust the owner. Rescue operations by an owner have raised questions on decentralised nature of the projects. Also, there is a possibility of compromise of an owner's private key.
+
+At present, there are various implementation ideas for handling governance. Adhering to a standard API would enable general tools (like EtherScan) to populate governance information, thus increasing transparency to users. This can result in a wide adoption for contract governance.
+
+## Specification
+
+<!--The technical specification should describe the syntax and semantics of any new feature. The specification should be detailed enough to allow competing, interoperable implementations for any of the current Ethereum platforms (go-ethereum, parity, cpp-ethereum, ethereumj, ethereumjs, and [others](https://github.com/ethereum/wiki/wiki/Clients)).-->
+
+#### Notes
+
+- The following specifications use syntax from Solidity 0.6.0 (or above)
+- Governor is the board member who has voting right.
+- Power represents the voting privilege of a governor.
+
+A contract that is compliant with `ERC-2767` shall implement the following interface:
+
+```solidity
+/// @title ERC-2767 Governance with Privileged Voting Rights
+/// @dev ERC-165 InterfaceID: 0x4fe54581
+interface ERC2767 {
+    /// @dev This emits when governor power changes
+    event GovernorPowerUpdated(address indexed governor, uint256 power);
+
+    /// @notice Gets the consensus power of the governor
+    /// @param _governor Address of the governor
+    /// @return The governor's voting power
+    function powerOf(address _governor) external view returns (uint256);
+
+    /// @notice Gets the sum of the power of all governors
+    /// @return Sum of the power of all governors
+    function totalPower() external view returns (uint256);
+
+    /// @notice Gets number votes required for achieving consensus
+    /// @return Required number of consensus votes
+    function required() external view returns (uint256);
+
+    /// @notice Updates governor statuses
+    /// @param _governor Governor address
+    /// @param _newPower New power for the governor
+    /// @dev Only Governance can call
+    function setGovernor(address _governor, uint256 _newPower) external;
```

Comment:
This function doesn't seem like it needs to be standardized.

Reply:
I agree, users should be able to customize the way they want to do this. Any UI tool wouldn't care about this method at all.

Reply:
I disagree. User interfaces and tools can be developed to work with any EIP-2767 contract to add new governors and set the power. Having this function makes that possible.